db/models.py

```diff
@@ -0,0 +1,31 @@
+from sqlalchemy import (
+    Column,
+    Integer,
+    String,
+    Date,
+    ForeignKey,
+)
+from sqlalchemy.orm import relationship
+
+from db.database import Base
+
+
+class Book(Base):
+    __tablename__ = "books"
+
+    id = Column(Integer, primary_key=True, index=True)
+    title = Column(String(255), unique=True, nullable=False)
+    summary = Column(String(1023))
+    publication_date = Column(Date, nullable=False)
+    author_id = Column(Integer, ForeignKey("author.id"), nullable=False)
+
+    author = relationship("Author", back_populates="books")
+
+
+class Author(Base):
+    __tablename__ = "author"
+
+    id = Column(Integer, primary_key=True, index=True)
+    name = Column(String(63), nullable=False, unique=True)

```

Comment:
The `name` field in the `Author` model is unique. This constraint will prevent two authors from having the same name, which might not be desirable in all cases. Ensure this aligns with your application's requirements.